scripts/requirements-extra.txt

```diff
@@ -1,4 +1,4 @@
-pygit2>=1.6.1
```

Comment:
please do not fix versions like this, if new version has issues better do for now `pygit2<=1.10`. Also update `scripts/requirements-doc.txt`.